2019-07-11 21:43:00,031 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244DD68>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ee5a28d18d16ce04f2436b6fa5d12331
2019-07-11 21:44:12,224 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 21:45:07,998 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 21:45:18,020 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 21:45:35,986 - INFO - register_us.py[:239] - WyndaeIkee938@yyw.link用户注册成功！
2019-07-11 21:45:40,001 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 21:45:44,170 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D128>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2ced1f2c076f24feeed4ab8a72070dfa
2019-07-11 21:45:45,170 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6B38>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2ced1f2c076f24feeed4ab8a72070dfa
2019-07-11 21:45:46,185 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474978>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2ced1f2c076f24feeed4ab8a72070dfa
2019-07-11 21:47:14,731 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-11 21:49:55,945 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-11 21:51:44,878 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 21:53:08,514 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 21:53:18,555 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 21:53:34,751 - INFO - register_us.py[:239] - DenacEmanuela591@beads.ren用户注册成功！
2019-07-11 21:53:37,138 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 21:53:40,342 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c68274e0c0136397bced9b57efb0efe9
2019-07-11 21:53:41,374 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474BA8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c68274e0c0136397bced9b57efb0efe9
2019-07-11 21:53:42,390 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c68274e0c0136397bced9b57efb0efe9
2019-07-11 21:55:40,257 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 21:56:31,174 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 21:56:39,948 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 21:56:49,967 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 21:57:03,577 - INFO - register_us.py[:239] - TrudyePlummera321@beads.ink用户注册成功！
2019-07-11 21:57:08,118 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 21:57:12,292 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a2b97b7a8ccdb104c50a268ae1a348ab
2019-07-11 21:57:13,308 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29228BAC8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a2b97b7a8ccdb104c50a268ae1a348ab
2019-07-11 21:57:14,308 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8518>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a2b97b7a8ccdb104c50a268ae1a348ab
2019-07-11 22:03:01,511 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-11 22:03:14,017 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-11 22:14:55,783 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:16:05,996 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 22:16:14,532 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:16:24,557 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:16:41,514 - INFO - register_us.py[:239] - ShellicKaciec379@beads.group用户注册成功！
2019-07-11 22:16:47,667 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:16:51,809 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6B38>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9add6c851cb11f3091f35f73bda5236f
2019-07-11 22:16:52,810 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9add6c851cb11f3091f35f73bda5236f
2019-07-11 22:16:53,826 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9add6c851cb11f3091f35f73bda5236f
2019-07-11 22:18:05,819 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:18:57,006 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:19:07,040 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:19:20,224 - INFO - register_us.py[:239] - GitelaGilberte281@beads.pub用户注册成功！
2019-07-11 22:19:27,303 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:19:31,476 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474A20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f623291e5875da6cb19ddfb3b1d35a99
2019-07-11 22:19:32,493 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922577F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f623291e5875da6cb19ddfb3b1d35a99
2019-07-11 22:19:33,509 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f623291e5875da6cb19ddfb3b1d35a99
2019-07-11 22:20:16,620 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-11 22:32:01,844 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:33:07,739 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:33:17,762 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:33:32,481 - INFO - register_us.py[:239] - GinnyeNevaa105@beads.wang用户注册成功！
2019-07-11 22:33:36,007 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:33:40,163 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A84A8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5ea4a15df9d018180f7a735cc8715ed3
2019-07-11 22:33:41,179 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5ea4a15df9d018180f7a735cc8715ed3
2019-07-11 22:33:42,195 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A518>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5ea4a15df9d018180f7a735cc8715ed3
2019-07-11 22:34:56,830 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:35:38,982 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 22:35:46,919 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:35:56,936 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:36:09,169 - INFO - register_us.py[:239] - ValenciaaReedc283@beads.link用户注册成功！
2019-07-11 22:36:15,048 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:36:18,253 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2924748D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/8d35cb7a1ec11a90bb3928bdef2dc8c7
2019-07-11 22:36:19,268 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A2CC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/8d35cb7a1ec11a90bb3928bdef2dc8c7
2019-07-11 22:36:20,269 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A2160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/8d35cb7a1ec11a90bb3928bdef2dc8c7
2019-07-11 22:38:16,114 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:39:11,526 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:39:21,556 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:39:34,868 - INFO - register_us.py[:239] - AbbieaKeshiaa943@beads.name用户注册成功！
2019-07-11 22:39:59,442 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:40:02,570 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922572B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/092ffdfbcecd24569b065bcdbd9d5714
2019-07-11 22:40:03,570 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D4E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/092ffdfbcecd24569b065bcdbd9d5714
2019-07-11 22:40:04,585 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/092ffdfbcecd24569b065bcdbd9d5714
2019-07-11 22:41:16,885 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-11 22:43:58,734 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:45:55,431 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 22:46:05,207 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:46:15,242 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:46:27,558 - INFO - register_us.py[:239] - DekecBryanta231@beads.kim用户注册成功！
2019-07-11 22:46:33,245 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:46:36,418 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8908>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b0134bf879d6735991b8bbe689449658
2019-07-11 22:46:37,450 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A89E8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b0134bf879d6735991b8bbe689449658
2019-07-11 22:46:38,481 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29228B7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b0134bf879d6735991b8bbe689449658
2019-07-11 22:48:17,745 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 22:50:10,041 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 22:50:18,788 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 22:50:28,823 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 22:50:42,735 - INFO - register_us.py[:239] - FeleceaOsmarre790@beads.red用户注册成功！
2019-07-11 22:50:48,230 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 22:50:51,351 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D7B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e234dd8c8a923f60f20a03aec67b8700
2019-07-11 22:50:52,367 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A0F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e234dd8c8a923f60f20a03aec67b8700
2019-07-11 22:50:53,383 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201AF60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e234dd8c8a923f60f20a03aec67b8700
2019-07-11 22:50:58,697 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-11 23:03:32,502 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-11 23:06:18,972 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-11 23:08:42,415 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:09:54,980 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 23:10:03,216 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:10:13,253 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:10:48,247 - INFO - register_us.py[:239] - OcieeHeidee680@anker.site用户注册成功！
2019-07-11 23:10:52,329 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:10:55,457 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257A58>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23ba8d00e98a7db399ef8a825ffcdf98
2019-07-11 23:10:56,461 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257BA8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23ba8d00e98a7db399ef8a825ffcdf98
2019-07-11 23:10:57,458 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257908>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23ba8d00e98a7db399ef8a825ffcdf98
2019-07-11 23:13:15,791 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:15:24,942 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:15:34,974 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:16:15,359 - INFO - register_us.py[:239] - LewcSchuylere024@anker.wiki用户注册成功！
2019-07-11 23:16:18,641 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:16:22,967 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6eda56ecc06b338f75d6ba2ddb44f027
2019-07-11 23:16:23,970 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257AC8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6eda56ecc06b338f75d6ba2ddb44f027
2019-07-11 23:16:24,971 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6E80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6eda56ecc06b338f75d6ba2ddb44f027
2019-07-11 23:18:00,978 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:19:42,394 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-11 23:19:51,761 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:20:01,797 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:20:14,804 - INFO - register_us.py[:239] - SandyaShelliee740@anker.work用户注册成功！
2019-07-11 23:20:17,047 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:20:21,233 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B66A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5438aae2ebcd742562ddd89f638cfa8f
2019-07-11 23:20:22,262 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5438aae2ebcd742562ddd89f638cfa8f
2019-07-11 23:20:23,280 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DDA0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5438aae2ebcd742562ddd89f638cfa8f
2019-07-11 23:21:58,967 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:23:15,504 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:23:25,535 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:23:40,133 - INFO - register_us.py[:239] - GoreeAnnisc350@shein.xin用户注册成功！
2019-07-11 23:23:42,159 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:23:46,299 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D3C8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/946ebab26e517e61320a82e7a1a5d997
2019-07-11 23:23:47,299 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/946ebab26e517e61320a82e7a1a5d997
2019-07-11 23:23:48,294 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D438>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/946ebab26e517e61320a82e7a1a5d997
2019-07-11 23:25:00,669 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-11 23:26:17,430 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:27:21,327 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:27:31,365 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:27:45,885 - INFO - register_us.py[:239] - DudaIvarc037@shein.wang用户注册成功！
2019-07-11 23:27:59,864 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:28:04,013 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A89B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5abe3e993deb65458dab3e85dc51518f
2019-07-11 23:28:05,045 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257DD8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5abe3e993deb65458dab3e85dc51518f
2019-07-11 23:28:06,077 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8DD8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5abe3e993deb65458dab3e85dc51518f
2019-07-11 23:30:53,177 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:32:44,300 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:32:54,347 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:33:12,040 - INFO - register_us.py[:239] - PansycLindseya673@shein.kim用户注册成功！
2019-07-11 23:33:16,176 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:33:20,339 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922578D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d8f8f4740d9fa84ee8bdcadfb28b9265
2019-07-11 23:33:21,347 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d8f8f4740d9fa84ee8bdcadfb28b9265
2019-07-11 23:33:22,361 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257978>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d8f8f4740d9fa84ee8bdcadfb28b9265
2019-07-11 23:34:31,457 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:35:22,290 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:35:32,309 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:35:45,189 - INFO - register_us.py[:239] - LamisaOakesc624@zaful.mobi用户注册成功！
2019-07-11 23:35:47,352 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:35:50,509 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ebba746b4408e8cc5b712570460de4dd
2019-07-11 23:35:51,521 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29228B160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ebba746b4408e8cc5b712570460de4dd
2019-07-11 23:35:52,557 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8F60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ebba746b4408e8cc5b712570460de4dd
2019-07-11 23:36:03,778 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-11 23:47:16,948 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:48:22,892 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:48:32,922 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:48:47,801 - INFO - register_us.py[:239] - IvancNakishaa724@zaful.biz用户注册成功！
2019-07-11 23:48:52,059 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:48:56,226 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2924741D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b2794fd730aea0479b0e49d1c346cfd7
2019-07-11 23:48:57,232 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2924740B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b2794fd730aea0479b0e49d1c346cfd7
2019-07-11 23:48:58,243 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474F60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b2794fd730aea0479b0e49d1c346cfd7
2019-07-11 23:53:40,292 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-11 23:55:45,410 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-11 23:57:14,373 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-11 23:58:21,948 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-11 23:58:32,034 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-11 23:58:48,034 - INFO - register_us.py[:239] - QuaidaLahelaa697@zaful.xyz用户注册成功！
2019-07-11 23:58:51,165 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-11 23:58:55,331 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201AF98>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4b5e0580d90a170280dd206a1bbca60d
2019-07-11 23:58:56,334 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D748>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4b5e0580d90a170280dd206a1bbca60d
2019-07-11 23:58:57,348 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D048>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4b5e0580d90a170280dd206a1bbca60d
2019-07-12 00:00:10,998 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:01:01,152 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:01:11,186 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 00:01:24,526 - INFO - register_us.py[:239] - ObothcChicke796@zaful.net.cn用户注册成功！
2019-07-12 00:01:37,923 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 00:01:42,073 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29228BD68>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f30dd3e0cf18385e1b47f6a2247d32cd
2019-07-12 00:01:43,076 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474B00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f30dd3e0cf18385e1b47f6a2247d32cd
2019-07-12 00:01:44,077 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474F98>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f30dd3e0cf18385e1b47f6a2247d32cd
2019-07-12 00:01:49,178 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 00:16:05,782 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 00:18:29,089 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:19:47,847 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:19:58,044 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 00:20:33,080 - INFO - register_us.py[:239] - TerracKarie087@zaful.work用户注册成功！
2019-07-12 00:20:45,652 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 00:20:54,738 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292257748>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ef01b4cef08bff5a060c260c0c75e40d
2019-07-12 00:20:55,837 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ef01b4cef08bff5a060c260c0c75e40d
2019-07-12 00:20:57,439 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8198>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ef01b4cef08bff5a060c260c0c75e40d
2019-07-12 00:21:07,157 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 00:33:17,470 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:36:23,794 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 00:38:30,086 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:39:30,315 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:39:40,349 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 00:39:54,872 - INFO - register_us.py[:239] - ShanieLakeshae951@zaful.fun用户注册成功！
2019-07-12 00:40:00,959 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 00:40:04,116 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474CC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c819a7d0a50309254f731a826e743668
2019-07-12 00:40:05,119 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A940>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c819a7d0a50309254f731a826e743668
2019-07-12 00:40:06,117 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292474908>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/c819a7d0a50309254f731a826e743668
2019-07-12 00:41:41,731 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:42:45,947 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 00:42:55,638 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:43:05,693 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 00:43:21,678 - INFO - register_us.py[:239] - FionneHeatha827@ninebot.tv用户注册成功！
2019-07-12 00:43:29,131 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 00:43:32,498 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922509E8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0e8d3781f4dbcf79e7acbc0e54500acd
2019-07-12 00:43:33,496 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292250240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0e8d3781f4dbcf79e7acbc0e54500acd
2019-07-12 00:43:34,494 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292250630>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0e8d3781f4dbcf79e7acbc0e54500acd
2019-07-12 00:45:08,544 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 00:49:01,041 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 00:50:11,683 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:51:00,610 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 00:51:08,323 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:51:18,355 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 00:51:47,319 - INFO - register_us.py[:239] - PrueaHarriettc785@ninebot.ltd用户注册成功！
2019-07-12 00:51:51,413 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 00:51:54,614 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/888dcb497fcfadefde72264ba3250a28
2019-07-12 00:51:55,619 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/888dcb497fcfadefde72264ba3250a28
2019-07-12 00:51:56,623 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/888dcb497fcfadefde72264ba3250a28
2019-07-12 00:53:55,047 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:56:20,584 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 00:58:18,569 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 00:59:41,830 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 00:59:51,861 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:00:08,083 - INFO - register_us.py[:239] - AlfredoeVarecke867@nineboy.com.cn用户注册成功！
2019-07-12 01:00:10,204 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:00:13,342 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244DEF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7e1196fe1404180f62cffc1ab41cf3ea
2019-07-12 01:00:14,349 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244DA90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7e1196fe1404180f62cffc1ab41cf3ea
2019-07-12 01:00:15,347 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292078908>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7e1196fe1404180f62cffc1ab41cf3ea
2019-07-12 01:02:06,005 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 01:02:22,910 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 01:13:37,819 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 01:15:16,111 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 01:15:26,134 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:15:44,263 - INFO - register_us.py[:239] - ImalacAbelarda570@nineboy.top用户注册成功！
2019-07-12 01:15:58,257 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:16:01,423 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229DE10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bf349538ba5f9adc927d5323e3e437a9
2019-07-12 01:16:02,430 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D438>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bf349538ba5f9adc927d5323e3e437a9
2019-07-12 01:16:03,425 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DF28>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bf349538ba5f9adc927d5323e3e437a9
2019-07-12 01:17:30,153 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 01:18:35,381 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 01:18:45,412 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:19:01,751 - INFO - register_us.py[:239] - DonaeKyme097@nineboy.pro用户注册成功！
2019-07-12 01:19:03,989 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:19:07,118 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032358>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3d02fd34a945004e709d260571af8125
2019-07-12 01:19:08,120 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3d02fd34a945004e709d260571af8125
2019-07-12 01:19:09,114 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292250400>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3d02fd34a945004e709d260571af8125
2019-07-12 01:20:48,313 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 01:20:54,217 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 01:32:20,429 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 01:36:14,317 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 01:39:06,095 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 01:39:19,045 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 01:39:29,079 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:39:48,293 - INFO - register_us.py[:239] - VophsieNadac692@nineboy.site用户注册成功！
2019-07-12 01:39:51,532 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:39:54,689 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8D30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4e2069619f010fd8289a45d7642bae53
2019-07-12 01:39:55,690 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A85F8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4e2069619f010fd8289a45d7642bae53
2019-07-12 01:39:56,690 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8E48>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/4e2069619f010fd8289a45d7642bae53
2019-07-12 01:41:10,800 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 01:42:07,312 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 01:42:17,353 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:42:30,515 - INFO - register_us.py[:239] - XavieracAnitac763@nineboy.vip用户注册成功！
2019-07-12 01:42:55,748 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:42:59,938 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032C18>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d44bfef0644d5168a8d0ffb15251ef36
2019-07-12 01:43:00,938 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DA20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d44bfef0644d5168a8d0ffb15251ef36
2019-07-12 01:43:01,955 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d44bfef0644d5168a8d0ffb15251ef36
2019-07-12 01:44:12,813 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 01:45:02,496 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 01:45:09,847 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 01:45:19,889 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 01:45:32,614 - INFO - register_us.py[:239] - AlemaZabdie896@nineboy.biz用户注册成功！
2019-07-12 01:46:07,748 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 01:46:10,947 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DB38>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d19f09f07e5e2fbcd8203a5ffbceb1d9
2019-07-12 01:46:11,947 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229D588>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d19f09f07e5e2fbcd8203a5ffbceb1d9
2019-07-12 01:46:12,948 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229D710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d19f09f07e5e2fbcd8203a5ffbceb1d9
2019-07-12 01:46:16,574 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 01:57:32,215 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 01:57:37,264 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 02:08:44,982 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 02:10:22,318 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 02:12:16,870 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 02:13:25,593 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 02:14:16,190 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 02:14:26,213 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 02:14:39,013 - INFO - register_us.py[:239] - MalaFairfaxc960@nineboy.net用户注册成功！
2019-07-12 02:14:47,284 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 02:14:51,455 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229D6A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e8ec5926c335dc648208624d3831fda0
2019-07-12 02:14:52,486 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244DA90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e8ec5926c335dc648208624d3831fda0
2019-07-12 02:14:53,487 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244D7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e8ec5926c335dc648208624d3831fda0
2019-07-12 02:16:31,962 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 02:17:18,803 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 02:17:39,448 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 02:17:49,488 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 02:18:12,743 - INFO - register_us.py[:239] - NoreenaeBrynna561@nineboy.cc用户注册成功！
2019-07-12 02:18:15,070 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 02:18:19,228 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DBA8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/22d8c621ba1a644aa2ca72eabf78215d
2019-07-12 02:18:20,229 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D668>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/22d8c621ba1a644aa2ca72eabf78215d
2019-07-12 02:18:21,229 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229DF28>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/22d8c621ba1a644aa2ca72eabf78215d
2019-07-12 02:18:26,497 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 02:28:33,436 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 02:38:37,185 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 02:48:41,230 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 02:59:24,621 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 03:09:26,672 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 03:19:48,418 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 03:29:55,826 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 03:40:03,388 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 03:50:06,465 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:00:27,660 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:10:32,289 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:20:41,246 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:30:53,217 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:40:55,715 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 04:50:59,371 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:01:01,558 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:11:05,057 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:21:25,578 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:31:29,218 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:41:32,060 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 05:52:03,900 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:02:06,399 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:12:16,808 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:22:44,316 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:32:46,986 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:43:07,930 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 06:53:10,194 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:03:12,115 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:13:32,574 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:23:35,401 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:33:38,261 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:43:42,368 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 07:53:50,025 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:04:05,998 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:14:09,153 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:24:30,315 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:34:34,206 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:44:42,598 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 08:54:58,102 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:05:06,901 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:15:10,367 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:25:21,573 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:35:24,744 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:46:01,545 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 09:56:05,540 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:06:09,127 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:15:12,200 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:25:23,434 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:35:25,557 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:45:28,216 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 10:55:30,785 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:05:37,111 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:15:40,955 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:25:46,648 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:35:55,824 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:46:03,903 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 11:56:05,760 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:06:11,026 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:16:20,340 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:26:23,386 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:36:25,775 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:46:28,008 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 12:56:33,305 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:06:38,649 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:16:40,881 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:26:44,677 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:36:48,693 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:46:51,051 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 13:56:55,456 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:07:04,895 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:17:08,129 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:27:10,627 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:37:19,317 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:47:21,424 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 14:57:27,471 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:07:36,177 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:17:38,621 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:27:41,260 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:37:45,464 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:47:59,279 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 15:58:14,626 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:08:19,080 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:18:21,640 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:28:45,820 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:38:59,809 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:49:29,302 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 16:59:31,848 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 17:09:34,630 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 17:19:43,330 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 17:29:47,466 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 17:39:51,856 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 17:49:59,404 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 18:00:18,628 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 18:10:22,308 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 18:22:03,135 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 18:23:16,194 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 18:24:14,938 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 18:24:24,985 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 18:24:38,146 - INFO - register_us.py[:239] - VallariaCosimac140@aukey.online用户注册成功！
2019-07-12 18:24:39,439 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 18:24:42,597 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96f90d78a1e83337ceecb11575b0e24b
2019-07-12 18:24:43,613 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2920320B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96f90d78a1e83337ceecb11575b0e24b
2019-07-12 18:24:44,630 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2920321D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96f90d78a1e83337ceecb11575b0e24b
2019-07-12 18:26:21,117 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 18:30:40,641 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 18:31:58,320 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 18:32:56,256 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 18:33:04,567 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 18:33:14,594 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 18:33:29,954 - INFO - register_us.py[:239] - HonoraSamuele972@aukey.live用户注册成功！
2019-07-12 18:33:34,967 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 18:33:39,941 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E0F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a3ff4b54396ccef4630832b92554542f
2019-07-12 18:33:41,240 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a3ff4b54396ccef4630832b92554542f
2019-07-12 18:33:42,300 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E518>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a3ff4b54396ccef4630832b92554542f
2019-07-12 18:36:06,506 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 18:37:22,429 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 18:37:32,472 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 18:37:47,660 - INFO - register_us.py[:239] - DeborahcMiaa134@aukey.fun用户注册成功！
2019-07-12 18:37:50,556 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 18:37:53,718 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E7B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/cabb769140f0d6e2986f76ba9edd3e36
2019-07-12 18:37:54,724 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229D0B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/cabb769140f0d6e2986f76ba9edd3e36
2019-07-12 18:37:55,726 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/cabb769140f0d6e2986f76ba9edd3e36
2019-07-12 18:39:41,865 - INFO - register_us.py[:150] - 等待超时，程序终止！
2019-07-12 18:40:45,899 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 18:41:42,654 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 18:41:52,698 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 18:42:05,288 - INFO - register_us.py[:239] - ParkeBarrettaa218@aukey.ltd用户注册成功！
2019-07-12 18:42:32,106 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 18:42:35,301 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29229D1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a0954d8a1871b3f38126dbc4108299e4
2019-07-12 18:42:36,302 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a0954d8a1871b3f38126dbc4108299e4
2019-07-12 18:42:37,301 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245EB38>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/a0954d8a1871b3f38126dbc4108299e4
2019-07-12 18:45:43,136 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 18:53:05,536 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 18:55:05,230 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 18:58:11,428 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 19:00:12,473 - INFO - register_us.py[:146] - 注册出错，程序结束！
2019-07-12 19:03:04,498 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:05:24,325 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:05:34,734 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:06:00,172 - INFO - register_us.py[:239] - KeshiaaEudoraa257@aukey.net.cn用户注册成功！
2019-07-12 19:06:08,800 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:06:13,383 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0024513708d2d10ba214b4e1496781d5
2019-07-12 19:06:14,407 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DFD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0024513708d2d10ba214b4e1496781d5
2019-07-12 19:06:15,415 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246D828>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/0024513708d2d10ba214b4e1496781d5
2019-07-12 19:10:00,467 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:15:11,948 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:15:21,985 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:15:41,523 - INFO - register_us.py[:239] - PompeyeHettiea937@aukey.mobi用户注册成功！
2019-07-12 19:15:44,713 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:15:48,041 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29245E080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6b07eb34ee812976720367cb267f9013
2019-07-12 19:15:49,073 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6b07eb34ee812976720367cb267f9013
2019-07-12 19:15:50,183 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8DA0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6b07eb34ee812976720367cb267f9013
2019-07-12 19:18:58,146 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:21:45,222 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 19:22:00,616 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:22:10,645 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:23:00,234 - INFO - register_us.py[:239] - PennicYanac579@nineboys.co用户注册成功！
2019-07-12 19:23:05,001 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:23:09,585 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29244D898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/217ca3d6f0862948f95eba0162ff68c7
2019-07-12 19:23:11,985 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29203C0B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/217ca3d6f0862948f95eba0162ff68c7
2019-07-12 19:23:13,094 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29203CB00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/217ca3d6f0862948f95eba0162ff68c7
2019-07-12 19:26:15,601 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:28:33,235 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 19:28:46,629 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:28:56,665 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:29:39,902 - INFO - register_us.py[:239] - WillaaMicke678@nineboys.store用户注册成功！
2019-07-12 19:29:41,565 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:29:44,776 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29203C7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6c8b28e9c3c8f9879936d762153d32eb
2019-07-12 19:29:45,774 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A7F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6c8b28e9c3c8f9879936d762153d32eb
2019-07-12 19:29:46,773 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29201A6A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6c8b28e9c3c8f9879936d762153d32eb
2019-07-12 19:31:05,040 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:34:54,722 - INFO - register_us.py[:217] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-12 19:35:32,791 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:35:42,825 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:36:23,164 - INFO - register_us.py[:239] - ClarkaPhilipae394@nineboys.art用户注册成功！
2019-07-12 19:37:08,565 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:37:12,743 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96b5516d4c0153ff179f64444beaf28f
2019-07-12 19:37:13,747 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29203C518>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96b5516d4c0153ff179f64444beaf28f
2019-07-12 19:37:14,752 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29203C0F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/96b5516d4c0153ff179f64444beaf28f
2019-07-12 19:38:23,464 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:39:17,064 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:39:27,169 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:39:41,209 - INFO - register_us.py[:239] - LaurentcAdie847@nineboys.site用户注册成功！
2019-07-12 19:39:42,837 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:39:45,982 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C29246DF60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/221095250e34548ef6e319d2675da5c8
2019-07-12 19:39:46,984 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6B00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/221095250e34548ef6e319d2675da5c8
2019-07-12 19:39:47,988 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922B6AC8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/221095250e34548ef6e319d2675da5c8
2019-07-12 19:41:02,143 - INFO - register_us.py[:156] - 正在设置用户的基本信息...
2019-07-12 19:41:59,687 - INFO - register_us.py[:233] - 点击提交个人信息的设置!
2019-07-12 19:42:09,722 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-12 19:42:47,216 - INFO - register_us.py[:239] - KurtisaSatchela720@nineboys.pro用户注册成功！
2019-07-12 19:42:49,154 - INFO - register_us.py[:273] - 注册信息已成功入库！
2019-07-12 19:42:53,742 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C292032080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b345de6f6ba05a217788cdb3e2625a37
2019-07-12 19:42:54,743 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8B00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b345de6f6ba05a217788cdb3e2625a37
2019-07-12 19:42:55,742 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000C2922A8898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/b345de6f6ba05a217788cdb3e2625a37
2019-07-12 19:42:59,675 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 19:53:05,546 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:03:08,968 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:13:11,997 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:23:29,360 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:33:32,507 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:43:34,834 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 20:53:38,890 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:04:26,401 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:14:30,729 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:24:34,525 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:34:36,461 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:45:30,199 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 21:55:35,449 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:06:30,438 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:16:37,470 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:26:46,503 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:37:31,988 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:47:35,847 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 22:57:46,239 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:08:29,087 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:18:50,789 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:28:57,195 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:39:29,156 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:49:44,428 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-12 23:59:58,702 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 00:10:09,949 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 00:20:25,793 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 00:30:43,629 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 00:40:46,268 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 00:51:15,600 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:01:20,763 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:11:28,029 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:21:37,488 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:31:40,766 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:41:43,187 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 01:51:46,843 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:03:23,677 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:13:29,702 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:23:36,694 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:33:28,711 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:44:27,266 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 02:54:37,126 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:05:35,367 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:18:04,214 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:28:30,976 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:38:39,052 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:48:54,355 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 03:59:24,553 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 04:09:32,882 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 04:19:38,475 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 04:29:53,636 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 04:40:06,779 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 04:50:11,764 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:00:31,347 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:10:34,174 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:20:38,581 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:30:44,065 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:40:48,580 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 05:50:58,988 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:01:20,494 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:11:28,691 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:21:34,904 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:31:41,755 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:44:16,691 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 06:56:40,373 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:06:45,779 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:17:26,408 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:28:18,567 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:38:21,726 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:48:25,428 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 07:58:31,053 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 08:08:51,137 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 08:18:53,401 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 08:29:11,406 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 08:39:43,948 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 08:49:50,730 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:01:27,564 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:13:01,116 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:23:17,058 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:33:19,557 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:43:25,151 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 09:53:33,464 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:03:35,978 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:13:38,211 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:23:40,226 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:33:44,100 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:43:57,165 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 10:54:32,788 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 11:04:35,114 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 11:14:42,396 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 11:24:46,379 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 11:34:49,722 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
2019-07-13 11:45:11,635 - INFO - register_us.py[:347] - 当前无注册任务，程序[sleep 10m]...
